drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyConnectionFactoryConfig.java

```diff
@@ -63,6 +64,19 @@ public DriftNettyConnectionFactoryConfig setConnectionPoolEnabled(boolean connec
         return this;
     }
 
+    @Min(1)
+    public int getConnectionPoolConcurrency()
+    {
+        return connectionPoolConcurrency;
+    }
+
+    @Config("thrift.client.connection-pool.concurrency")
```

Comment:
This isn't so much concurrency as it is connections per endpoint. I would definitely rename it because it suggests something like # of threads or limit on # of threads when in reality it's just how many connections are created to each endpoint.